event-config.h.cmake

```diff
@@ -36,394 +36,400 @@
 #define EVENT__PACKAGE_TARNAME ""
 
 /* Define if libevent should build without support for a debug mode */
-#cmakedefine EVENT__DISABLE_DEBUG_MODE
+#cmakedefine EVENT__DISABLE_DEBUG_MODE 1
```

Comment:
Do we really need this change?

Reply:
This was part of a bigger checkout from the master branch instead of 2.1.8.